2021/41xxx/CVE-2021-41043.json

```diff
@@ -4,14 +4,63 @@
     "data_version": "4.0",
```

Comment:
('Expecting property name enclosed in double quotes: line 64 column 13 (char 1900)',)